spec/fixtures/locales/fr_valid.yml

```diff
@@ -8,3 +8,4 @@ fr_valid:
       other: "bars"
     full_key: "baz"
     interpolation: "%{qux}"
+    ignored_key: "rofl"
```

Comment:
The missing interpolation variable test doesn't (as I first expected) test translation files against the ruby source. Instead, it checks the non-primary locale translation files against the primary locale's. Because of that, this key will produce the error, since it doesn't contain the `%{quz}` interpolation present in the English translation.